pkg/store/dashboard/dashboard.go

```diff
@@ -1,7 +1,101 @@
 package dashboard
 
+import (
+	"github.com/dwarvesf/fortress-api/pkg/model"
+	"gorm.io/gorm"
+)
+
 type store struct{}
 
 func New() IStore {
 	return &store{}
 }
+
+func (s *store) GetWorkSurveysByProjectID(db *gorm.DB, projectID string) ([]*model.WorkSurvey, error) {
+	var rs []*model.WorkSurvey
+
+	query := `
+	select feedback_events.end_date, AVG((Case
+		WHEN "order" = 1 THEN ( (
+			 Case
+				  When length(answer) = 0 THEN null
+				  ELSE cast(answer AS integer)
+			 END
+			 )
+			)
+		 END)) as workload,
+		 AVG((Case
+		WHEN "order" = 2 THEN ( (
+			 Case
+				  When length(answer) = 0 THEN null
+				  ELSE cast(answer AS integer)
+			 END
+			 )
+			)
+		 END)) as deadline,
+		 AVG((Case
+		WHEN "order" = 3 THEN ( (
+			 Case
+				  When length(answer) = 0 THEN null
+				  ELSE cast(answer AS integer)
+			 END
+			 )
+			)
+		 END)) as learning
+	 
+		 from feedback_events left join employee_event_topics eet on feedback_events.id = eet.event_id
+		 left join employee_event_questions eeq on feedback_events.id = eeq.event_id
+		 where eet.project_id= ?
+			   group by eeq.event_id, feedback_events.id
+			order by feedback_events.end_date desc
+			limit 6
+	`
+
+	return rs, db.Debug().Raw(query, projectID).Scan(&rs).Error
+}
+
+func (s *store) GetAllWorkSurveys(db *gorm.DB) ([]*model.WorkSurvey, error) {
+	var rs []*model.WorkSurvey
+
+	query := `
```

Comment:
format it

Reply:
updated

Reply:
remove `(` `)` unnecessary, put the work of the same level in the same column, and uppercase the words like `CASE`, `WHEN`,...